Resources/Prototypes/_NF/Recipes/Cooking/meal_recipes.yml

```
@@ -184,6 +184,20 @@
   - Oven
   - Microwave
 
+# Cured Meat - Future category for sausages/transformative meat and now consumption friendly.
+
+- type: microwaveMealRecipe
+  id: RecipeCuredCarp
+  name: purified carp 
```

```suggestion
  name: cured carp
```
Cured's probably fine.